src/markdown/annual-reports/TwentyTwenty.yaml

```diff
@@ -25,51 +25,51 @@ departmentSection:
               subtitle: User Testing
 timelineSection:
     pretitle: Milestones
-    title: New Faces & Birthrays
-    subtitle: The team gets stronger each year as we embrace new additions, celebrate anniversaries, and gain more experience.
-    designers:
-        - designer:
-              url: wood-jon
-              startDate: 2019-04-15
+    title: Significant Markers in a Marked Year
+    events:
+        - event:
+              detail: birthRay
+              designer: wood-jon
+              startDate: 2020-04-15
```

Comment:
just as an FYI I'm going to refactor this component to use our "author" object so we don't have to add the startDate, name, location, etc